### DISCUSSION TITLE
Sharing GeMS-related scripts

### DISCUSSION THREAD

**Discussion by alwunder** [usgs/gems-tools-pro#55] [Scripts]
As promised at DMT'23 I'm going to "practice what I preach" and give it a go... sharing Python scripts that I have developed to help doing GeMS database work.

I have posted a few scripts in the Issues section.  It's not much, but it's a start.  Let me and others know if this is a good idea by sharing your comments.

Just trying to get the ball rolling here.  As others chime in, I hope that this initiative takes shape.  It may eventually move to another platform (like a Hub site) as I discussed at DMT'23 with Evan and others.  But let's see what we can do here!

**Comment by ethoms-usgs** [comment]
Yeah, I was just looking more closely at the Discussions section and I think this is a good place for stuff like this. Let's keep Issues related to existing tools in the toolbox. I added a Discussion Category called Scripts and moved your three over here.

**Reply by alwunder** [reply]
Yes, agreed!  This is great!  Thanks for moving the stuff around and reclassifying it.  I've got a few more things I think I'd like to share and will do so through Discussions.  I think with this setup, we're in a good place for folks to take a look and comment.  I've already let a few folks who were interested in the code know I've posted some to the GeMS GitHub.  I think I'll also let Dave know and perhaps we can send a note thought the Listserv to let everyone know where to look/post if they have things they'd like to share.
